backends/sycl-ref/ceed-sycl-ref-basis.sycl.cpp

```diff
@@ -656,5 +655,4 @@ int CeedBasisCreateH1_Sycl(CeedElemTopology topo, CeedInt dim, CeedInt num_nodes
   CeedCallBackend(CeedSetBackendFunctionCpp(ceed, "Basis", basis, "Destroy", CeedBasisDestroyNonTensor_Sycl));
   return CEED_ERROR_SUCCESS;
 }
-
```

Comment:
Whitespace changes like this were made for consistency as well. This sort of change in the sycl, sycl-ref, and sycl-shared folders should be undone